cnames_active.js

```diff
@@ -148,6 +148,7 @@ var cnames_active = {
   "arcn": "alexakasanjeev.github.io/arcn",
   "argo": "albertosantini.github.io/argo", // noCF? (don´t add this in a new PR)
   "ari": "arbo77.github.io/ari",
+  "ariang: "p3terx.github.io/ariang",
```

Comment:
```suggestion
  "ariang": "p3terx.github.io/ariang",
```
Missing closing quote